serveradmin/__init__.py

```diff
@@ -2,5 +2,3 @@
 
 Copyright (c) 2020 InnoGames GmbH
 """
-
-VERSION = (1, 9, 3)
```

Comment:
Shouldn't it be `from adminapi import VERSION`?

Reply:
This would be possible too. I decided to adjust all current imports so that there is only one origin to check and one does not have to go along a chain of imports.